pubsub/include/marlin/pubsub/PubSubNode.hpp

```diff
@@ -865,8 +925,11 @@ int PUBSUBNODETYPE::did_recv_message(
 		// MESSAGE
 		case 3: return this->did_recv_MESSAGE(transport, std::move(bytes));
 		break;
+		// RECEIPT
+		case 4: return this->did_recv_RECEIPT(transport, std::move(bytes));
```

Comment:
Keep HEARTBEAT at 4, use a different number if you want. In general, avoid changing message numbers both for backward compatibility and our own code which might rely on the numbers.